src/alttxt/generator.py

```diff
@@ -28,34 +29,83 @@ def __init__(
         # L0 and L1 don't care about sort/aggregation
 
         if self.level == Level.ONE:
-            text_desc += self.descriptions["level_1"][self.verbosity.value]
+            text_desc += self.descriptions["level_1"]["upset_introduction"]
+            text_desc += self.descriptions["level_1"]["dataset_properties"]
 
         elif self.level == Level.TWO:
-            # L2 splits generation by sort type of the plot
-            text_desc += self.descriptions["level_2"]\
-                    [self.verbosity.value][self.grammar.sort_by]
+            text_desc += self.descriptions["level_2"]["set_description"]
+            text_desc += self.descriptions["level_2"]["intersection_description"]
+            text_desc += self.descriptions["level_2"]["statistical_information"]
             
         elif self.level == Level.DEFAULT:
             # Default level is combination of L1 and L2
-            text_desc += self.descriptions["level_1"][self.verbosity.value]
+
+            # A short alternative text description
+            altText = self.descriptions["AltText"]
+
+            # A short description of the technique used in plot visualization
+            technique = self.descriptions["level_1"]["upset_introduction"]
+
+            # Structured text starts here
+            introduction = self.descriptions["level_1"]["upset_introduction"]
+            text_desc += introduction
             text_desc += " "
-            text_desc += self.descriptions["level_2"]\
-                    [self.verbosity.value][self.grammar.sort_by]
+
+            dataset_properties = self.descriptions["level_1"]["dataset_properties"]
+            text_desc += dataset_properties
+            text_desc += " "
+            
+            set_description = self.descriptions["level_2"]["set_description"]
+            text_desc += set_description
+            text_desc += " "
+
+            intersection_description = self.descriptions["level_2"]["intersection_description"]
+            text_desc += intersection_description
+            text_desc += " "
+
+            statistical_information = self.descriptions["level_2"]["statistical_information"]
+            text_desc += statistical_information
+
+            if self.structure:
+            # Construct the dictionary for markdown content
+                data_to_write_as_md = {
+                    "UpSet Introduction": self.replaceTokens(introduction),
+                    "Dataset Properties": self.replaceTokens(dataset_properties),
+                    "Set Properties": self.replaceTokens(set_description),
+                    "Intersection Properties": self.replaceTokens(intersection_description),
+                    "Statistical Information": self.replaceTokens(statistical_information)
+                }
+
+                markdown_content = ""
+                for section, content in data_to_write_as_md.items():
+                    markdown_content += f"# {section}\n{content}\n\n"
```

Comment:
`section` here looks like `upset_description`. We need to change it so that it looks like `Upset Description`. This will require a little bit of string manipulation